dell-csi-helm-installer/verify.sh

```diff
@@ -370,15 +370,24 @@ function verify_authorization_proxy_server() {
     WGET=$(ssh ${NODEUSER}@"${node}" "which wget")
     if [ -x "${WGET}" ]; then
       log info "Running wget on "${node}""
+      nr=1
       if [ "${insecure}" == "true" ]
       then
         resp=$(ssh ${NODEUSER}@"${node}" wget --no-check-certificate --server-response --spider --quiet https://"${proxyHost}" 2>&1)
```

Comment:
why not set log level to get rid of warning line , then you dont need to look for second line in output ?
ssh -o LogLevel=error xxx

Reply:
I'm not sure if that affects any of the other responses. This way seems the least intrusive.